.github/cla.md

```diff
@@ -0,0 +1,23 @@
+## Contributor License Agreement
+The following terms are used throughout this agreement:
+
+* You - the person or legal entity including its affiliates asked to accept this agreement. An affiliate is any entity that controls or is controlled by the legal entity, or is under common control with it.
+* Project - is an umbrella term that refers to any and all Mojira open source projects.
+* Contribution - any type of work that is submitted to a Project, including any modifications or additions to existing work.
+* Submitted - conveyed to a Project via a pull request, commit, issue, or any form of electronic, written, or verbal communication with IOMED Medical Solution S.L., contributors or maintainers.
+
+## 1. Grant of Copyright License.
+
+Subject to the terms and conditions of this agreement, You grant to the Projects’ maintainers, contributors, users and to Mojira a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable copyright license to reproduce, prepare derivative works of, publicly display, publicly perform, sublicense, and distribute Your contributions and such derivative works. Except for this license, You reserve all rights, title, and interest in your contributions.
+
+## 2. Grant of Patent License.
+
+Subject to the terms and conditions of this agreement, You grant to the Projects’ maintainers, contributors, users and to Mojira a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable (except as stated in this section) patent license to make, have made, use, offer to sell, sell, import, and otherwise transfer your contributions, where such license applies only to those patent claims licensable by you that are necessarily infringed by your contribution or by combination of your contribution with the project to which this contribution was submitted.
+
+If any entity institutes patent litigation - including cross-claim or counterclaim in a lawsuit - against You alleging that your contribution or any project it was submitted to constitutes or is responsible for direct or contributory patent infringement, then any patent licenses granted to that entity under this agreement shall terminate as of the date such litigation is filed.
+
+## 3. Source of Contribution.
+
+Your contribution is either your original creation, based upon previous work that, to the best of your knowledge, is covered under an appropriate open source license and you have the right under that license to submit that work with modifications, whether created in whole or in part by you, or you have clearly identified the source of the contribution and any license or other restriction (like related patents, trademarks, and license agreements) of which you are personally aware.
+
+_Based in [GitHub's CLA](https://cla.github.com/agreement)__
```

Comment:
1. This is a dead link
2. "Based in" should be "Based on"
3. There is an underscore too much at the end